src/GUIPlayer.java

```diff
@@ -14,8 +22,26 @@ public GUIPlayer(){
 	@Override
 	public boolean rollAgain(int totalSoFar) {
 		System.out.println("Score is "+myScore+ " Round score is "+totalSoFar);
+		/* 
+		 * add new code; use if statement to stop the game when the score of the Human player (player1) 
+		 * achieves or exceeds the wining score (100)
+		 * */
+		if (isHumanPlayerScoreReachWin(totalSoFar)) {
```

Comment:
This fix does not go here.  There is already a method to check for a win.  It is the buggy code, and that is what you should fix.